### DISCUSSION TITLE
Auto-load Previews - External app

### DISCUSSION THREAD

**Discussion by danielnc** [lookbook-hq/lookbook#122] [Q&A]
Is there a way to embed all polaris components in my lookbook preview on my own app?

I have new components and styled components from polaris and I want to make sure I see them side by side and use lookbook as a way to document all components usage

Without copying the preview files I can't manage make lookbook auto-load the previews that exists within this gem

**Comment by allmarkedup** [comment]
Hi @danielnc, sorry for the super-slow reply, I was away on holiday and just got back.

So as I understand it, you are pulling in the Polaris components as a gem, and you want to see the previews of them in Lookbook alongside your own bespoke components. Is that correct?

If I've understood that correctly, then you *may* be able to do something like this:

```rb
# config/application.rb
polaris_gem_dir = Gem::Specification.find_by_name("polaris_view_components").gem_dir
config.lookbook.preview_paths << "#{polaris_gem_dir}/demo/test/components/previews"
```

I've not tested this yet so YMMV but let me know how you get on if you can :-)
